lblod/app-lokaal-mandatenbeheer
# lmb-1028: create migration to set gemeenteraadslid to lid in rangorde

## Description

We change the rangorde label of GR to be lid instead of gemeenteraadslid. Create a migration to update all

## How to test

- In IV create some mandatarissen to have a rangorde _gemeenteraadslid_ of _Gemeenteraadslid_ 
- Let the migrations run
- See if they are updated (restart cache and resource)
